test/broker_test/broker_test.go

```diff
@@ -571,6 +572,25 @@ var _ = test.DescribeTestsFor(test.TestCase{
 						Expect(transitiveResourcesActualCount).To(Equal(transitiveResourcesExpectedCount))
 					})
 				})
+
+				Context("when broker is responding slow", func() {
+					It("should timeout", func() {
+						brokerServer.CatalogHandler = func(rw http.ResponseWriter, req *http.Request) {
+							rw.WriteHeader(http.StatusOK)
+							if fl, ok := rw.(http.Flusher); ok {
+								for i := 0; i < 30; i++ {
```

Comment:
I see we get very innovative with our testing as time goes on :D

Reply:
Will this one get in unnoticed :D

Reply:
noticed ! :)